src/components/Breadcrumbs.tsx

```diff
@@ -3,16 +3,15 @@ import {
   Breadcrumbs as Mui_Breadcrumbs,
   BreadcrumbsProps as Mui_BreadcrumbsProps,
   Container,
-  Link,
+  Link as MuiLink,
```

Comment:
Can we make this consistent with an underline, as e.g. Mui_BreadcrumbsProps please